### ISSUE TITLE
normalizePath error

### ISSUE THREAD

**Issue by summerfallusa** [rstudio/reticulate#1545]
I think this is related to https://github.com/rstudio/reticulate/issues/1542, https://github.com/rstudio/reticulate/issues/1176 and https://github.com/rstudio/reticulate/issues/1460.  I used anaconda instead of miniconda. Otherwise, I think the issues are the same. 

I installed anaconda under my WLS2 - ubuntu 20.04. 
"which conda" showed that the binary conda is in folder: "/home/myname/anaconda3/condabin/conda"   instead of  "/home/summer/anaconda3/bin/conda"  

After starting R, I could see the list of environments, but couldn't use the environment 
```
> library(reticulate)
> conda_list()
             name                                                 python
1            base                      /home/summer/anaconda3/bin/python
2          ml_env          /home/summer/anaconda3/envs/ml_env/bin/python
3 my_r_python_env /home/summer/anaconda3/envs/my_r_python_env/bin/python
> use_condaenv("my_r_python_env", required = TRUE)
Error in normalizePath(conda, winslash = "/", mustWork = TRUE) : 
  path[1]="/home/summer/anaconda3/bin/conda env": No such file or directory
```
Even if I specify conda path as below it did not help (got the same  normalizePath error as above):
``` 
use_condaenv(conda = "/home/myname/anaconda3/condabin/conda", condaenv = "/home/summer/anaconda3/envs/my_r_python_env", required = TRUE)
```
In R, conda_binary points to "/home/summer/anaconda3/bin/conda" instead "/home/summer/anaconda3/condabin/conda", see below: 
```
> conda_binary()
[1] "/home/summer/anaconda3/bin/conda"
```
Below are py_config info in case it helps (note that this is not in the target environment "my_r_python_env"): 
```
> py_config()
python:         /home/summer/.virtualenvs/r-reticulate/bin/python
libpython:      /usr/lib/python3.8/config-3.8-x86_64-linux-gnu/libpython3.8.so
pythonhome:     /home/summer/.virtualenvs/r-reticulate:/home/summer/.virtualenvs/r-reticulate
version:        3.8.10 (default, Nov 22 2023, 10:22:35)  [GCC 9.4.0]
numpy:          /home/summer/.virtualenvs/r-reticulate/lib/python3.8/site-packages/numpy
numpy_version:  1.24.4

Below are session Info in case it helps: 
> sessionInfo()
R version 4.3.3 (2024-02-29)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: Ubuntu 20.04.6 LTS

Matrix products: default
BLAS/LAPACK: /home/summer/anaconda3/envs/my_r_python_env/lib/libopenblasp-r0.3.26.so;  LAPACK version 3.12.0

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        LC_COLLATE=C.UTF-8    
 [5] LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8    LC_PAPER=C.UTF-8       LC_NAME=C             
 [9] LC_ADDRESS=C           LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

time zone: America/New_York
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] reticulate_1.35.0

loaded via a namespace (and not attached):
 [1] compiler_4.3.3    Matrix_1.6-5      cli_3.6.2         tools_4.3.3       withr_3.0.0       rstudioapi_0.15.0
 [7] rappdirs_0.3.3    Rcpp_1.0.12       grid_4.3.3        jsonlite_1.8.8    rlang_1.1.3       png_0.1-8        
[13] lattice_0.22-5  

```

**Comment by TXMUNC** [comment]
Not sure why a string " env" was added to path[1], as indicated in"path[1]="/home/summer/anaconda3/bin/conda env": No such file or directory"

**Comment by t-kalinowski** [comment]
Fixed by #1545
